web/api/v1/api.go

```diff
@@ -371,6 +378,27 @@ func (api *API) query(r *http.Request) (result apiFuncResult) {
 	if res.Err != nil {
 		return apiFuncResult{nil, returnAPIError(res.Err), res.Warnings, qry.Close}
 	}
+	var ranker Ranker = sum{}
+
+	if res.Value.Type() == parser.ValueTypeVector {
+		vec, ok := res.Value.(promql.Vector)
+		if !ok {
+			err := errors.New("Cannot convert to vector")
```

Comment:
Nit: We start error message with a smaller case letter but logs with upper case letters, unless for acronyms :) (same below)